scripts/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Template build script
+# ---------------------
+
+rm -rf CMakeCache.txt CMakeFiles
+
+export CXX=FIXME
+export MKLROOT=FIXME
+HPX_ROOT=FIXME
+BLASPP_ROOT=FIXME
+LAPACKPP_ROOT=FIXME
+
+# CMAKE_BUILD_TYPE := Release | RelWithDebugInfo | Debug
+#
+# DLAF_WITH_CUDA   := ON | OFF (default)
+#
+# DLAF_WITH_MKL    := ON | OFF (default)
+#
+# BUILD_DOC        := ON | OFF (default)
+#
+# DLAF_WITH_TEST   := ON (default) | OFF 
+#
+cmake ~/code/DLA-Future \
+  -D CMAKE_BUILD_TYPE=RelWithDebugInfo \
+  -D CMAKE_PREFIX_PATH="$HPX_ROOT;$BLASPP_ROOT;$LAPACKPP_ROOT" \
+  -D DLAF_WITH_MKL=ON \
```

Comment:
I would try to be as close as possible to the cmake command line help.

```bash
  cmake [options] <path-to-source>
  cmake [options] <path-to-existing-build>
  cmake [options] -S <path-to-source> -B <path-to-build>
```
So, I would put options first and the paths after.